src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/client/CcdClient.java

```diff
@@ -30,6 +39,97 @@
         return caseResponse.getData();
     }
 
+    public void startRejectEventAndSubmit(
+        String accessToken,
+        String s2sToken,
+        String userId,
+        String ccdId,
+        Container container
+    ) {
+        Map<String, Object> caseData = getCaseData(accessToken, s2sToken, ccdId);
```

Comment:
not sure I can use this in submit?

Reply:
No need to fetch data just use the case data from start event